client/go/internal/cli/cmd/login.go

```diff
@@ -87,6 +88,10 @@ This command runs a browser-based authentication flow for the Vespa Cloud contro
 			if err != nil {
 				// log the error but move on
 				cli.printWarning("Could not store the refresh token locally. You may need to login again once your access token expires")
+				cli.printWarning("To persist the refresh token using file storage (unencrypted), set VESPA_CLI_DUMMY_KEYRING=1 environment variable")
```

Comment:
Could we add explicit switch for this rather than environment variable? Scary if someone puts this in their `.bashrc` and forgets about it.

I would also perhaps give this a separate sentence on the security aspect instead of just saying "(unencrypted)" saying something like "Note: Storing the refresh token unencrypted on your file system means someone with access to this file can get unauthorized access to your application for the life of the refresh token (24 hours)".

...or something a bit shorter.

Reply:
Could we also link to some Vespa CLI documentation giving more information about the secret store implementation and debug advice?